README.md

```diff
@@ -126,7 +126,7 @@ $ ybdata validate fixtures/mydata
 This should print out a table of both required and optional items for validation, and the validation status should be listed at the bottom. Once validated, convert the dataset into a package:
 
 ```
-$ ybdata convert fixtures/mydata
+$ ybdata package fixtures/mydata
```

Comment:
I'll admit it's been a while since I used this tool - I assume this is a typo and it should have been "package" all along?

Reply:
Yep, I believe so. The README is very thorough so I was able to infer from the description.